### ISSUE TITLE
Start updates discussion in open-app Loomio group

### ISSUE THREAD

**Issue by simontegg** [open-app/holodex#58]
I'm feeling blocked on this because I haven't seen much visible progress on other tasks that we've collectively agreed are important.

**Comment by simontegg** [comment]
A lack of visible progress on tasks does not give me much confidence for fronting any comms. Particularly given recent history.

**Comment by ahdinosaur** [comment]
between [Craftworks](http://craftworks.enspiral.com) facilitation, leading a push for [NodeSchool Wellington](https://github.com/nodeschool/wellington), helping bring [Cobudget](http://cobudget.co) dev back to life, and [learning how to crawl](https://github.com/open-app/holodex/issues/56), i've been juggling a bit, but still trudging forward. :smile_cat: i'm trying my best and for once i feel happy, so not sure what else to do. how do you feel @simontegg?

**Comment by simontegg** [comment]
The issue is that people (both internal and external) will unavoidably overlay expectations onto any comm unless the comms are made super explicit. So if the comms says "Mikey will work fulltime on holodex.", they will overlay an expectation of what that means based on their own experience.

For most people, this means "Mikey will treat Holodex as his full time job" -> which means "all kinds of tasks that aren't necessarily fun will be completed based on someone with a high degree of stakeholding (essentially a "founder") toiling away" -> which means "As a stakeholder, I can expect regular, significant improvement to the value I receive from Holodex". These expectations form in other's minds, but since that is completely natural I believe its the responsibility of the _communicator_ to mitigate thee formation of these perhaps unrealistic expectations. 

This situation does not appear to be the case. Initiating but not meeting expectations is a major anti-pattern both internally and externally (IMO).

If we communicate our commitment accurately we can avoid this and reach a good state, perhaps even a virtuous cycle. At the moment because there is a vicious cycle where the uncertainty created by poor communication makes me feel reluctant to commit to Holodex. -> "Who will do all the necessary but not so fun tasks? Me?"

**Comment by ahdinosaur** [comment]
cool, thanks @simontegg for sharing. :+1: 

so is your perception that i'm not committed to the necessary but not so fun tasks? the only task i'm focused on at the moment is https://github.com/open-app/holodex/issues/56 (except minor issue management and loomio comms edits). i understand if i haven't produced much in the way of visible output from my work, but i'd much rather we as a team to value effort over output.

if the problem is feeling like another team member isn't putting in the effort they committed to, maybe time tracking would be a great indicator for our effort (or lack thereof) and possibly help with this problem, or am i misinterpreting the problem? admittedly this week has been hectic for me, feeling like i'm constantly context switching, but at least you'd be able to have visibility into that, plus it'd help me measure and experiment with how to organize my work life so i'm putting in the hours i want into Holodex.

**Comment by simontegg** [comment]
How we value contribution is a separate issue. The main issue here is that I don't understand the shape of your commitment to Holodex and I don't want stakeholders to form unrealistic expectations of the value they should receive from Holodex. Allowing these expectations to form and then *failing to meet them* has significant negative implications for team Holodex's *reputation*. 

What is the shape of your commitment to Holodex?
What kinds of work do you wish to engage in? Code, comms, bizdev, data entry? What are most/least passionate about? What is the balance between these?  What kind of work mode will you bring to the work? Exclusively hammock-driven? Get-shit-done? What is the balance between different modes?

In the past 2.5 months you've fixed some bugs, but haven't produced much visible value? Has something(s) been blocking that? Are there pre-conditions that are not being met?  Do you anticipate these conditions changing?

If our situation is "conditions have to be just right before Mikey engages in work and when he does so it is exclusively in a hammock-driven mode" How will this affect *fairness* (if other members are working in [less privileged modes](http://blog.loomio.org/2014/01/28/pattern-recognition/) ?

A lack of clarity around these questions creates a lot of uncertainty. This uncertainty is the main driver behind my poor ratings of our *reputation*, *communication* and *fairness*.

I feel blocked on any comms work because I continue to perceive our *reputation* *communication* and *fairness* as poor.

**Comment by ahdinosaur** [comment]
i'll admit i'm finding these recent conversations to be personally upsetting, as this highlights a major problem i feel within Enspiral where our standards for our work are so high that we're either stressed for working so hard or miserable because we failed to achieve the absolute best at everything. in this atmosphere of soul crushing pessimism, i struggle to feel good about myself, as it seems at every moment i'm only failing over and over and over again. i'd rather we do more to celebrate our success, i reckon Holodex has had a lot of success this last quarter and i'm grateful for where we're at, thanks @simontegg for being with me on this journey.

anyways,

> What is the shape of your commitment to Holodex?

my primary focus is Holodex, i've already dropped my commitment to other projects like Cobudget in order to allow for this. i intend to put in sustained effort on Holodex, for the immediate future in a primary project capacity (4 days per week). Holodex is the first necessary ingredient in my omega project so i'm here for the long haul, it's already been a few years in progress and i'm happy to continue the slow and steady march towards technical and cultural infrastructure for holons.

> What kinds of work do you wish to engage in? Code, comms, bizdev, data entry? What are most/least passionate about? What is the balance between these? What kind of work mode will you bring to the work? Exclusively hammock-driven? Get-shit-done? What is the balance between different modes?

i'm keen to do what needs to get done, but it's important not to confuse intention and effort for productivity and output. i intend to put as much effort as i can muster, but i'm a human who makes mistakes and have a personal life (or lack thereof) that adds additional randomness. when i do work, i have wild variance in productivity depending on what i'm doing and how i'm feeling, sometimes difficult tasks become painless and sometimes easy tasks become impossible, regardless of how much i bang my head against the work. sometimes this can lead to positive or negative feedback cycles that i don't always realize are happening, i'm still learning about myself and how to work better.

> In the past 2.5 months you've fixed some bugs, but haven't produced much visible value? Has something(s) been blocking that? Are there pre-conditions that are not being met? Do you anticipate these conditions changing?

i'm a 22-year-old human, not a machine. conditions always change, if you feel i haven't done anything valuable then maybe my next dice rolls will yield something that meets your anticipated targets.

> If our situation is "conditions have to be just right before Mikey engages in work and when he does so it is exclusively in a hammock-driven mode" How will this affect fairness (if other members are working in less privileged modes ?

if that's how you perceive how i work, i'm sorry. i think it's unfair for any of our team to be working in a poor environment, i'd rather we value the shared happiness of our team, where we can each work in a abundant situation we feel enthusiastic about, than compete in a scarcity race to the bottom. it's obviously not always possible, we do have to do work we don't always feel great about, but at least then i wish we would celebrate success than beat each other up over failure.


> I feel blocked on any comms work because I continue to perceive our reputation communication and fairness as poor.

i support you, feel free to do what you think is best for you. you're well-deserving of a break anyways, especially with your recent intake of other work.

@simontegg, i reckon we should catch up in person to continue this conversation.
